redis/internal/redisprom/pools.go

```diff
@@ -3,12 +3,15 @@ package redisprom
 import (
 	"github.com/prometheus/client_golang/prometheus"
 	"github.com/prometheus/client_golang/prometheus/promauto"
+
```

Comment:
and please don't add the extra blank line, which would cause sorting issues :)

Reply:
go fmt just re-adds it every time

Reply:
I'm pretty sure that's `goimport` not `go fmt` (a lot of IDEs uses `goimport` instead of `go fmt`)

Reply:
sure either way it just comes back on its own using standard tooling

Reply:
`goimport` is not "standard tooling" :)